submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx

```diff
@@ -1,10 +1,236 @@
+      submittedDate: "2021-10-11",
+      submittedBy: "Jane Doe",
+      isCompleted: false,
+    },
+    {
+      id: 5,
+      name: "Management Plan 123",
+      submittedDate: "2021-10-10",
+      submittedBy: "John Doe",
+      isCompleted: true,
+    },
+    {
+      id: 6,
+      name: "Management Plan 456",
+      submittedDate: "2021-10-11",
+      submittedBy: "Jane Doe",
+      isCompleted: true,
+    },
+    {
+      id: 7,
+      name: "Management Plan 123",
+      submittedDate: "2021-10-10",
+      submittedBy: "John Doe",
+      isCompleted: true,
+    },
+    {
+      id: 8,
+      name: "Management Plan 456",
+      submittedDate: "2021-10-11",
+      submittedBy: "Jane Doe",
+      isCompleted: false,
+    },
+  ];
+
+  return (
+    <Container maxWidth="xl" sx={{ mt: 2 }}>
+      <Paper
+        sx={{
+          width: "90%",
+          height: "100%",
+          borderRadius: "6px",
+        }}
+        elevation={2}
+      >
+        <Box
+          bgcolor={BCDesignTokens.themeBlue10}
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "flex-start",
+            borderRadius: "3px 3px 0 0",
+          }}
+          height={HEADER_HEIGHT}
+        >
+          <Typography
+            variant="h3"
+            fontWeight={600}
+            px={2}
+            color={BCDesignTokens.typographyColorPrimary}
+          >
+            Copper Mine
+          </Typography>
+        </Box>
+
+        <Box
+          sx={{
+            padding: "36px 12px 12px 12px",
+          }}
+        >
+          <Box
+            sx={{
+              borderRadius: "3px",
+              border: `1px solid #F2F2F2`,
+              boxShadow: "0px 1px 2px rgba(0, 0, 0, 0.1)",
+            }}
+          >
+            <Box
+              display={"flex"}
+              justifyContent={"space-between"}
+              sx={{ py: BCDesignTokens.layoutPaddingXlarge }}
+            >
+              <CardInnerBox>
+                <Typography variant="h4" fontWeight={400}>
+                  Management Plans
+                </Typography>
+                <ProjectStatus bold status={PROJECT_STATUS.POST_DECISION} />
+              </CardInnerBox>
+              <CardInnerBox>
+                <Button>
+                  <AddIcon sx={{ p: 0, mr: 0.5 }} />
+                  New Submission
+                </Button>
+              </CardInnerBox>
+            </Box>
+            <Box height={"100%"} px={BCDesignTokens.layoutPaddingXsmall}>
+              <Divider sx={{ mb: 0.5 }} />
+              <Typography
+                variant="body1"
+                sx={{
+                  fontWeight: "bold",
+                  backgroundColor: BCDesignTokens.themeGold10,
+                }}
+              >
+                Active Submissions
+              </Typography>
+              <CardInnerBox sx={{ height: "100%" }}>
+                <TableContainer component={Box} sx={{ height: "100%" }}>
```

Comment:
why not all of this is in the SubmissionTable component

Reply:
because the tables are sharing the same headers in the design

Reply:
nvm i found a fix